src/old-components/Guilds/ActionsBuilder/SupportedActions/SetPermissions/SetPermissionsInfoLine.tsx

```diff
@@ -0,0 +1,50 @@
+import { useMemo } from 'react';
+import { ActionViewProps } from '..';
+import { Segment } from '../common/infoLine';
+import Avatar from 'old-components/Guilds/Avatar';
+import { FiArrowRight, FiNavigation } from 'react-icons/fi';
+import useENSAvatar from 'hooks/Guilds/ether-swr/ens/useENSAvatar';
+import { MAINNET_ID, shortenAddress } from 'utils';
+import { ParsedDataInterface } from './types';
+
+// TODO: What is the most logical way to present the information? Is more information needed?

```

Comment:
Nice todo, there's quite a lot we can do here and I'll make some follow up issues for it